docs/getting_started.md

```diff
@@ -24,17 +24,17 @@ This is a guide on how to get started with CAPV (Cluster API Provider vSphere).
 
 #### clusterctl
 
-Please download the latest `clusterctl` from the Cluster API (CAPI), GitHub [releases page](https://github.com/kubernetes-sigs/cluster-api/releases).
+Download the latest `clusterctl` from the Cluster API (CAPI), GitHub [releases page](https://github.com/kubernetes-sigs/cluster-api/releases).
 
-#### Docker
+##### Docker
 
 Docker is required for the bootstrap cluster using `clusterctl`. See the [docker documentation](https://docs.docker.com/glossary/?term=install) for install instructions.
 
-#### Kind
+##### Kind
```

Comment:
Same here

Reply:
Ok , fixed indentation w/ kind being under clusterctl and others being top level

Reply:
ok... i kept this indent...   but removed the others